src/donut/system.cljc

```diff
@@ -113,6 +113,12 @@
   (and (= ::ref (ref-type ref))
        (= 1 (count (ref-key ref)))))
 
+(defn component-id [ref]
```

Comment:
What do you think of this?

```clojure
(defn component-id [ref]
  (->> ref
       ref-key
       (take 2)
       (vec)))
```

It might be a little less performant but my guess is that it wouldn't be significant